src/main/java/frc/robot/commands/ArmDefaultCommand.java

```diff
@@ -0,0 +1,56 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants.ArmConstants;
+import frc.robot.Constants.OperatorConstants;
+import frc.robot.subsystems.ArmSystem;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+
+public class ArmDefaultCommand extends CommandBase {
+    // Goal is to get joystick input and turn that into what we want to set the arm to
+    private XboxController m_Controller;
+    private ArmSystem m_arm = new ArmSystem();
+
+    public ArmDefaultCommand(ArmSystem arm) {
+        m_arm = arm;
```

Comment:
addRequirements(m_arm)

Reply:
Good point @gsimonnet!  I spoke with David, and he showed me that the issue is fixed in develop.  Let us know if you're OK resolving this comment.